pkg/ansible/proxy/proxy_test.go

```diff
@@ -56,7 +55,15 @@ func TestHandler(t *testing.T) {
 		t.Fatalf("Error starting proxy: %v", err)
 	}
 
-	po := createPod("test", "default", mgr.GetConfig())
+	cl, err := client.New(mgr.GetConfig(), client.Options{})
+	if err != nil {
+		t.Fatalf("Error to create the client: %v", err)
```

Comment:
```suggestion
		t.Fatalf("Failed to create the client: %v", err)
```

Ditto for the `createPod` error.